src/windows-emulator/syscalls.cpp

```diff
@@ -1919,44 +1898,45 @@ namespace
 		return STATUS_NOT_SUPPORTED;
 	}
 
+	// TODO: warning stdcall problem
 	NTSTATUS WINAPI handle_NtQuerySymbolicLinkObject(const syscall_context& c, const handle link_handle,
```

Comment:
i think that's just a copy-paste issue on my end. stdcall/WINAPI can safely be removed here

Reply:
removed